src/apps/dictionaries/components/DictionaryForm.tsx

```diff
@@ -71,6 +71,20 @@ const useStyles = makeStyles({
   }
 });
 
+const supportedLocalesLabel = (values: any) => {
```

Comment:
For Source details also, we have the supported locale field. Can we make it common and use for both dictionaries and sources.

Reply:
@kirity-mahanty Added the same function for sources as well, but making it common doesn't seem reasonable as this function is returning a jsx element will have to make it as a component., as it is returning a group of material ui component